### ISSUE TITLE
`GradientColorStops` also support opacity suffixes

### ISSUE THREAD

**Issue by francoismassart** [francoismassart/eslint-plugin-tailwindcss#68]
`GradientColorStops` also support opacity suffixes, @francoismassart.

```diff
- if (['backgroundColor', 'textColor', 'placeholderColor'].includes(propName)) {
+ if (['backgroundColor', 'textColor', 'placeholderColor', 'gradientColorStops'].includes(propName)) {
```

_Originally posted by @smhmd in https://github.com/francoismassart/eslint-plugin-tailwindcss/pull/53#discussion_r751700335_

**Comment by francoismassart** [comment]
I believe this should be fixed in version 3.0.0

If you are still using TailwindCSS v2 then a fix could be release later under major version 2

**Comment by francoismassart** [comment]
Does anyone need this for v2 or you all updated to TailwindCSS c3?

**Comment by smhmd** [comment]
Version 3 🚀

**Comment by francoismassart** [comment]
already fixed in v3 and will be fixed in next release 2.0.0 for tw2 users
